Assets/MRTK/SDK/Features/UX/Scripts/Pointers/BaseControllerPointer.cs

```diff
@@ -671,6 +679,6 @@ public override void OnInputDown(InputEventData eventData)
             }
         }
 
-        #endregion  IMixedRealityInputHandler Implementation
```

Comment:
Nit: a bunch of whitespace looks like it was changed for some reason...

Reply:
Fixing....